src/components/InfoBanner/Layout.js

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+import { colorLightBlue, colorWhite, fontSmall, spacingMedium } from '../../styles/designTokens';
+
+const Layout = styled.div`
```

Comment:
I would call it Wrapper :)
Also there is Content component which is generic responsive wrapper, which you could put it inside it:
```
function InfoBanner({ children }) {
  return (
    <Wrapper role="banner"> // Just layout / wrapper with bgr color
      <Content>{children}</Content>
    </Wrapper>
  )
}
```